updating/index.adoc

```diff
@@ -0,0 +1,85 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
+[id='updating-clusters-overview-understand-update-service']
```

Comment:
In v4.8 [doc](https://docs.openshift.com/container-platform/4.8/updating/understanding-openshift-updates.html), there is not `Understanding the OpenShift Update Service`.

Reply:
Addressed

Reply:
ack